pkg/apis/eventing/v1alpha1/channel_defaults_test.go

```diff
@@ -16,10 +16,86 @@ limitations under the License.
 
 package v1alpha1
 
-import "testing"
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+var (
+	defaultChannelProvisioner = &corev1.ObjectReference{
```

Comment:
`defauktClusterChannelProvisioner` ?

Reply:
I prefer the existing name, but as this name is only used within the tests, I don't have a strong opinion. 

You are correct that the kind is a `ClusterChannelProvisioner` today, but I think we intend to add namespaced `ChannelProvisioner`s in the future, which would be valid in this ref. So in my mind this is the Channel's provisioner, regardless of its kind, which is why I like this name.